roles/sap_hana_install/tasks/pre_install/hdblcm_prepare.yml

```diff
@@ -58,7 +72,9 @@
                    done | awk '$2=="{{ ansible_architecture }}"{print $1}'
                 ); do
            ( printf "%s " ${sapcar_matching_arch}
-             ./${sapcar_matching_arch} --version |
+             cp ${sapcar_matching_arch} {{ sap_hana_install_software_extract_directory }} &&\
+             chmod 755 {{ sap_hana_install_software_extract_directory }}/${sapcar_matching_arch}
```

Comment:
We can also run a chmod root:root here so we do not need to call the file module later.